### PR TITLE
Az ob steps

### PR DESCRIPTION
**Pull request by lbliii**
wasn't able to use tabs -- this theme doesnt have the shortcode. I re-structured the headers to separate these two cloud guides.

### CODE REVIEW

**Comment by rclagett** [comment]
Hey @CryptoPishka I think you were reviewing the wrong page. The content your making comments about is located here https://deploy-preview-215--metriclyhelpsite.netlify.app/integrations/microsoft-azure/azure-cli-installation/ while we're updating this https://deploy-preview-215--metriclyhelpsite.netlify.app/getting-started/onboarding-wizard/.

**Comment by CryptoPishka** [comment]
oh deer , my fault... reviewing the correct one

@rclagett do you think my comments are still valid for https://deploy-preview-215--metriclyhelpsite.netlify.app/integrations/microsoft-azure/azure-cli-installation/ ?

**Comment by CryptoPishka** [comment]
my correct comments:

Copy and paste the appId into the Client ID field. - make appId 'copiable' to clipboard

the rest looks correct

File: content/getting-started/onboarding-wizard.md [hunk 1]
```diff
@@ -88,13 +91,99 @@ If the **Report Path Prefix** lookup is not available (and is instead a freeform
...
+8. Log in to your account and return to the command line.
+9. Input the following command:
+`az account show`
+10. Locate your **tenantId** and copy it.
+11. Paste the Id into the **Tenant Id** field.
+![tenant-id](/images/onboarding-wizard/tenant-id.png)
+12. Select **Continue**.
+13. Run the following command, adding in your id for **subscription-id**:
+`az ad sp create-for-rbac --role "Monitoring Reader" --name CloudWisdomReader --scopes /subscriptions/<subscription-id>`
+14. Copy and paste the `appId` into the **Client ID** field.
```
Comment (rclagett) [inline content/getting-started/onboarding-wizard.md:126]: To keep consistency with earlier in the doc:
```suggestion
14. Copy and paste the **appId** into the **Client ID** field.
```

File: content/getting-started/onboarding-wizard.md [hunk 2]
```diff
@@ -88,13 +91,99 @@ If the **Report Path Prefix** lookup is not available (and is instead a freeform
...
+15. Copy and paste the `password` into the **Access Key** field.
```
Comment (rclagett) [inline content/getting-started/onboarding-wizard.md:127]: To keep consistency with earlier in the doc:
```suggestion
15. Copy and paste the **password** into the **Access Key** field.
```

File: content/getting-started/onboarding-wizard.md [hunk 3]
```diff
@@ -88,13 +91,99 @@ If the **Report Path Prefix** lookup is not available (and is instead a freeform
...
+   - **Name**: `cloudwisdom-integration`
+   - **Supported account types**: Accounts in this organizational directory only - Single tenant
+   - **Redirect URI (optional)**: `https://us.cloudwisdom.virtana.com`
+8. **Azure Tab**: Select **Register**. You are re-routed to your new apps overview page.
+![azure-setup-portal](/images/onboarding-wizard/azure-setup-portal.png)
+9. **CloudWisdom Tab**: Select **Continue**.
+10. **Azure Tab**: Copy and paste the following into the CloudWisdom's Onboarding Wizard:
+   - **Application (client) ID**: maps to Client ID
+   - **Directory (tenant) ID**:  maps to Tenant ID
+![az-port-gif](/images/onboarding-wizard/az-port-gif.gif)
```
Comment (rclagett) [inline content/getting-started/onboarding-wizard.md:152]: Would it be terribly difficult to trim the top and bottom of the gif so as not to see the tabs and task bar?

**Comment by rclagett** [comment]
> oh deer , my fault... reviewing the correct one
> 
> @rclagett do you think my comments are still valid for https://deploy-preview-215--metriclyhelpsite.netlify.app/integrations/microsoft-azure/azure-cli-installation/ ?

Yes I do think they are valid. Were you thinking of creating a separate PR?

File: content/getting-started/onboarding-wizard.md [hunk 4]
```diff
@@ -88,13 +90,99 @@ If the **Report Path Prefix** lookup is not available (and is instead a freeform
...
```
Comment (rclagett) [inline content/getting-started/onboarding-wizard.md:151]: Would it be terribly difficult to trim the top and bottom of the gif so as not to see the tabs and task bar?

Comment (lbliii) [reply content/getting-started/onboarding-wizard.md]: cropped!

File: content/getting-started/onboarding-wizard.md [hunk 5]
```diff
@@ -88,13 +94,98 @@ If the **Report Path Prefix** lookup is not available (and is instead a freeform
...
+6. Open Windows PowerShell.
+7. Input the following command:
+`az login`
+8. Log in to your Azure account.
+9. Run the following command: `az account show`
+10. Locate **id** and **tenantId** in the output.
+11. Copy and paste the **tenantId** value into the **Tenant Id** field in CloudWisdom.
+![tenant-id](/images/onboarding-wizard/tenant-id.png)
+12. Select **Continue**.
+13. Return to PowerShell and run the following command, replacing *<subscription-id>* with the **id** value:
```
Comment (rclagett) [inline content/getting-started/onboarding-wizard.md:126]:
```suggestion
13. Return to PowerShell and run the following command, replacing *\<subscription-id>* with the **id** value:
```

**Pull request merged**